api/nakadi-event-bus-api.yaml

```diff
@@ -278,6 +278,9 @@ paths:
 
          * Events in the stream may be visible for a short period of time before being removed.
 
+         On nakadi-live, the oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
```

Comment:
I don't think we should mix open-source stuff with our private operational information. "nakadi-live" doesn't say anything for people outside Zalando.